association_test.R

```diff
@@ -0,0 +1,62 @@
+#! /usr/bin/env Rscript
+library(argparser)
+library(magrittr)
+argp <- arg_parser("Run association test") %>%
+  add_argument("gds_file", help = "GDS file") %>%
+  add_argument("pheno_file",
+               help = "Phenotype file in annotated dataframe format") %>%
+  add_argument("grm_file", help = "GRM file") %>%
+  add_argument("outcome", help = "Outcome variable name") %>%
+  add_argument("family", help = "Distribution family",
+               default = "gaussian") %>%
+  add_argument("--out_file", help="output file name",
+               default = "assoc.rds") %>%
+  add_argument("--covars",
+               help = "Covariate variable names (space-separated)") %>%
+  add_argument("--variant_id", help = "File with vector of variant IDs") %>%
+  add_argument("--sample_id", help = "File with vector of sample IDs")
+
+argv <- parse_args(argp)
+
```

Comment:
I realized that if you move the library calls after all of the argparser stuff, it's much faster to run `foo.R --help`

Reply:
I didn't think of that! Can you make the same change in the other scripts?

Reply:
Yeah, sure thing